cmp/tab/TabContainerModel.js

```diff
@@ -87,6 +94,21 @@ export class TabContainerModel {
 
             this.forwardRouterToTab(this.activeTabId);
         }
+
+        if (track) {
+            this.addReaction({
+                track: () => this.activeTab,
+                run: (activeTab) => {
+                    const route = activeTab.containerModel.route;
+                    XH.track({
+                        msg: `Viewed ${activeTab.title}`,
+                        category: 'Tab',
```

Comment:
I am sure this is what we had in Toolbox, but having 'Tab' as the category seems a bit awkward now that we're building it in.  How about:
```
{category: 'Navigation', message: `Viewed ${activeTab.title} tab`, ....}
```

Also note that `msg` is the old param that we're trying to stamp out - trackService supports it as a fallback, but let's swap to `message` which we're looking to use consistently on the client.